### PR TITLE
add missing regex to our CVE indicator

### PR DESCRIPTION
**Pull request by avidan-H**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/21361

## Description
Regex was missing from our CVE indicator. Copied the regex from the deprecated version of the CVE indicator.

## Required version of Demisto
5.0.0

## Does it break backward compatibility?
   - No

## Must have
- [x] Tests
- [ ] Documentation (with link to it)
- [ ] Code Review

## Dependencies
Mention the dependencies of the entity you changed as given from the precommit hooks in checkboxes, and tick after tested them.
- [ ] Dependency 1

## Technical writer review
Mention and link to the files that require a technical writer review.
- [ ] [reputations_CHANGELOG.md](https://github.com/demisto/content/edit/update-cve-indicator/Misc/reputations_CHANGELOG.md?pr=%2Fdemisto%2Fcontent%2Fpull%2F5312)
- [ ] [reputation-cve_CHANGELOG.md](https://github.com/demisto/content/edit/update-cve-indicator/Misc/reputation-cve_CHANGELOG.md?pr=%2Fdemisto%2Fcontent%2Fpull%2F5312)

### CODE REVIEW

File: Misc/reputations.json [hunk 1]
```diff
@@ -124,18 +124,6 @@
...
-      "version": -1,
-      "regex": "CVE-\\d{4}-\\d{4,7}",
-      "reputationScriptName": "cveReputation",
-      "details": "CVE CVSS Score",
-      "contextValue": "ID",
-      "contextPath": "CVE(val.ID && val.ID === obj.ID)",
-      "enhancementScriptNames": [
-        "cveSearch"
-      ]
-    },
```
Comment (bakatzir) [inline Misc/reputations.json]: the `Misc/reputations.json` is for prior to 5.0 instances. why did you delete it?

Comment (avidan-H) [reply Misc/reputations.json]: Because it was decided that indicator type `CVE CVSS Score` can be removed completely. The question is if we remove this old one completely then do we need to update the `fromVersion` field of the new indicator type `CVE` to reach to older server versions as well. Ani mivarer achshav.

Comment (avidan-H) [reply Misc/reputations.json]: Spoke to @JonathanMeler. We are going to add `"toVersion": 4.9.9` to the old indicator type instead of deleting or disabling it.

File: Misc/reputation-cve.json [hunk 2]
```diff
@@ -7,11 +7,11 @@
   "commitMessage": "",
   "shouldPublish": false,
   "shouldCommit": false,
-  "regex": "",
+  "regex": "CVE-\\d{4}-\\d{4,7}",
   "details": "CVE",
   "prevDetails": "CVE",
-  "reputationScriptName": "",
-  "reputationCommand": "",
+  "reputationScriptName": "cveReputation",
```
Comment (avidan-H) [inline Misc/reputation-cve.json]: I'm not sure whether this is necessary or relevant - I saw it on the old indicator so I added it. @bakatzir do you know?

**Comment by avidan-H** [comment]
@kirbles19 can you take a look at the changelog files linked above in the description?

File: Misc/reputation-cve.json (hunk 2 above)
Comment (bakatzir) [reply Misc/reputation-cve.json]: this automation is not a formatting script as unescapeips, what it does is run the `cve-search` command. which is a hack. we should implement in this integration a `cve` command instead.

Comment (bakatzir) [reply Misc/reputation-cve.json]: If you wish to only merge this indicator. you can leave this for now, and open an issue to implement a `cve` comamnd instead in all the relevant integrations(which include `CVE Search` and `Kenna v2` AFAIK).

Comment (avidan-H) [reply Misc/reputation-cve.json]: There's already an issue for that.

**Comment by kirbles19** [comment]
@avidan-H Done
